spec/dummy/config/database.yml

```diff
@@ -0,0 +1,22 @@
+# SQLite version 3.x
+#   gem install sqlite3
```

Comment:
I wonder do we need all of this here, like rails server config, assets etc. if we are only using this as library

Reply:
or is it actually used in offcents-api by active admin? I mean anything past Survey service

Reply:
https://guides.rubyonrails.org/engines.html#testing-an-engine

Reply:
👍